src/implementations/twig/storybook/package.json

```diff
@@ -15,10 +15,10 @@
     "@babel/preset-env": "7.11.0",
     "@babel/preset-react": "7.10.4",
     "@babel/runtime": "7.11.2",
-    "@storybook/addon-cssresources": "6.0.22",
-    "@storybook/addon-knobs": "6.0.22",
-    "@storybook/addon-viewport": "6.0.22",
-    "@storybook/html": "6.0.22",
+    "@storybook/addon-cssresources": "6.0.26",
```

Comment:
Why upgrading these at this pull request?

Reply:
well, i wanted to fix the problem we have with the text knob in the current next-v3 branch, basically, i noticed that when we had two different major version of storybook in react and twig for some reason this triggers a "bug" and it breaks if you try to access the knobs. Then i noticed that the version was upgradable (0.22 to 0.26) and i took the chance. I can put it back to 0.22, in both the packages here but at the same time why not updating them?

Reply:
No problem making an upgrade, sure. My thoughts were rather into the major bump in the [react one](https://github.com/ec-europa/europa-component-library/pull/1698/files#diff-74dfbd6d83f634d50878093b347f827aL19). First reaction was that we need to double-check more carefully after such a bump. But the second reaction is that react is going away, so no issue making whatever there.
If the updates are fixing the issue with knobs, good, keep the update pls

Reply:
ok, it's fine with 6.0.22, also the private packages have a dep on storybook addons and it's set on 6.0.22. The react private packages are not aligned, instead, but as you said this will disappear together with react, so i'm not touching them, knobs are working in the preview.